src/main/kotlin/racingcar/ui/InputView.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.ui
+
+object InputView {
+    fun inputValues(): Pair<Int, Int> {
+        val carCount = getNumber(inputCarCount())
+        val moveCount = getNumber(inputMoveCount())
+        return Pair(carCount, moveCount)
```

Comment:
이미 아실 수도 있지만 Pair 객체는 `to infix 함수`로도 간편하게 표현할 수 있어요. 😃
```suggestion
        return carCount to moveCount
```